packages/utils/path.ts

```diff
@@ -13,22 +13,51 @@ export function basename(path: string) {
 	return s[s.length - 1];
 }
 
+export const normalize = (path: string) => {
+	const parts = path.split('/');
```

Comment:
At present, `normalize` doesn't handle Windows paths (e.g. `C:\Users\Test\Documents`). This might be okay in `InteropService_Importer_Md` since `toForwardSlashes` is called before `normalize`. However,
- Documentation may help prevent bugs related to using `normalize` on Windows. For example, perhaps `normalize` could be named `normalizePosix`.
- `normalize` is called from `filename`. Is `filename` provided Windows-style paths in other parts of the code? If so, consider updating `filename` to support Windows-style paths.